backend/eventloop/application_event_loop/application_event_runner_syncruns.go

```diff
@@ -394,7 +394,7 @@ func (a *applicationEventLoopRunner_Action) applicationEventRunner_handleSyncRun
 			return false, err
 		}
 
-		// TODO: GITOPSRVCE-67 - DEBT - Include test case to check that the various goroutines are terminated when the CR is deleted.
+		// TODO: GITOPSRVCE-166 - DEBT - Include test case to check that the various goroutines are terminated when the CR is deleted.
```

Comment:
According to the story criteria [GITOPSRVCE-166](https://issues.redhat.com/browse/GITOPSRVCE-166 ) is about implementing sync operation in the backend. Is the test case in TODO part of the same story?

Reply:
Added it, thanks!